api/build.gradle

```diff
@@ -1,12 +1,10 @@
-import io.swagger.codegen.DefaultGenerator
-import io.swagger.codegen.config.CodegenConfigurator
 import org.pmiops.workbench.tooling.GenerateAPIListingTask
 import org.pmiops.workbench.tooling.IncrementalHotSwapTask
 
 // Runs before all tasks. Sets up properties and dependencies for the build
 // process itself.
 buildscript {
-  // External propertiges on the default project. Values declared in ext blocks
+  // External properties on the default project. Values declared in ext blocks
   // outside of the buildscsript block aren't usable here.
   ext {
     ELASTICSEARCH_VERSION = '6.8.3'
```

Comment:
aside: I never quite grokked what needs to go here vs. an external properties file.